packages/builders/src/interactions/slashCommands/Assertions.ts

```diff
@@ -15,12 +15,16 @@ export function validateName(name: unknown): asserts name is string {
 }
 
 const descriptionPredicate = s.string.lengthGe(1).lengthLe(100);
+const localePredicate = s.nativeEnum(Locale);
 
 export function validateDescription(description: unknown): asserts description is string {
 	descriptionPredicate.parse(description);
 }
 
 const maxArrayLengthPredicate = s.unknown.array.lengthLe(25);
+export function validateLocale(locale: unknown): asserts locale is LocaleString {
+	localePredicate.parse(locale);
```

Comment:
Uh, actually this should return what the predicate returns and use that because it can take an enum key and convert it to a value 😅